ci/roles/build_containers/defaults/main.yaml

```diff
@@ -39,7 +39,7 @@ release: 9
 image_tag: ''
 
 # Image Prefix
-container_name_prefix: openstack
+container_name_prefix: podified
```

Comment:
Why use podified? I would suggest we keep openstack. I don't like the term podified anyway.